docs/site/Understanding-the-differences.md

```diff
@@ -0,0 +1,106 @@
+---
+lang: en
+title: 'Understanding the differences between LoopBack 3 and LoopBack 4'
+keywords: LoopBack 4.0, LoopBack 4, LoopBack 3
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Understanding-the-differences.html
+---
+
+This article will help existing users understand LoopBack 4:
+
+- How to connect LoopBack 3.x concepts to LoopBack 4 terms
+- What's new and exciting in LoopBack 4
```

Comment:
While we're in this page.. I'm thinking whether we should have "what's new and exciting in LB4" (this bullet and the section at the bottom), because the list keeps changing.
I'm happy to open another issue to discuss this, and not blocking the landing of this PR.

Reply:
I added [this](https://github.com/strongloop/loopback-next/pull/3052/commits/bdacc61f140369a55ac38b2f9cd1135a30abb6c7#diff-46c6d1cbe6310da7865410e1888a47b3R107) but let me know if I should just remove the whole section instead.

Reply:
TBH, I personally prefer to have it completely. Let's keep it this way for now and we can improve the docs later. Thanks.